source/server/http_test_server_filter.h

```diff
@@ -18,7 +18,11 @@ class HttpTestServerDecoderFilterConfig : public FilterConfigurationBase {
 public:
   HttpTestServerDecoderFilterConfig(const nighthawk::server::ResponseOptions& proto_config);
 
-  std::shared_ptr<const nighthawk::server::ResponseOptions> getServerConfig();
+  /**
+   * @return std::shared_ptr<const nighthawk::server::TimeTrackingConfiguration> the base
```

Comment:
It may not be clear to the reader what "base configuration" is. Can we clarify that this is the startup configuration as read from the bootstrap which then later gets updated via in-flight headers?

Reply:
Done. Language changed across all three filters